_includes/header.html

```diff
@@ -5,7 +5,9 @@
                 <i class="fa fa-bars"></i>
             </button>
             <a class="navbar-brand page-scroll" href="/#page-top">
-                <span class="logo">{% include img/opensourcedesign-logo.svg %}</span>
+                <span class="logo">
+                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" alt="Open Source Design Logo"/>
```

Comment:
We don’t need this alt text, as the logo is purely cosmetic. It says "Open Source Design" in the next line, so we can use `alt=""` here. :)

Reply:
I don't know about the 'need', but it's a standard/accessibility practice to provide Alts for every image.

Reply:
Yes it is, but it's also accessibility practice to not duplicate text, and to use _empty_ alt text for these images. :) (Alt tag present but being empty like `alt=""`

See:

> Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page (alt="" or just alt - e.g. <img src="prettyswirl.jpg" alt>). If, for example, the information in a graph is explained in text below it, there’s no need to duplicate that in alt text.

From https://a11yproject.com/posts/alt-text/

Reply:
Created similar change for the JOBS repo at https://github.com/opensourcedesign/jobs/pull/320/files